core/domain/rights_manager.py

```diff
@@ -483,6 +787,19 @@ def can_modify_roles(self, activity_type, activity_id):
         return self.is_admin() or self.is_owner(activity_type, activity_id)
 
     def can_release_ownership(self, activity_type, activity_id):
+        """Returns whether this user has release ownership right for given
```

Comment:
... this user is allowed to release ownership of the given activity to the community.